pkg/app/ops/planpreviewoutputcleaner/cleaner.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package planpreviewoutputcleaner
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/pipe-cd/pipe/pkg/filestore"
+)
+
+const (
+	outputTTL = 48 * time.Hour
+	interval  = 24 * time.Hour
+	prefix    = "command-output/"
+)
+
+type Cleaner struct {
+	lister  filestore.Lister
+	deleter filestore.Deleter
```

Comment:
not sure why do we have to separate those interfaces. How about using `backend filestore.Store` 🤔

Reply:
Nice catch. 👍

Reply:
Fixed this. The whole `filestore.Store` was not used to avoid using unexpected functions like `Put` in this cleaner.